### ISSUE TITLE
Why does it use android write_external_storage permission

### ISSUE THREAD

**Issue by brianle5113** [mapsplugin/cordova-plugin-googlemaps#779]
Hi, I was wondering why does this plugin use android.permission.WRITE_EXTERNAL_STORAGE? Some of my testers expressed worry about it and I can't see the reason my self. Could some one let me know, thanks.﻿

**Comment by Hirbod** [comment]
Regarding to this
https://developers.google.com/maps/documentation/android-api/config

it was required some time ago. Since SDK 8.3 it should not be important anymore. I will have a look on that later

**Comment by Hirbod** [comment]
removed from master

**Comment by brianle5113** [comment]
Thanks for your quick reply and action it's really appreciated. Unfortunately even though the permission is no longer showing up AndroidManifest.xml or plugin.xml, my app still asks for read/write permission when I go to install it. 
I removed all my other plugins to make sure it wasn't one of them and still the same. I also created a completely new app and the only change I made was to install the plugin from git and still it asks for read/write to sd card permission.
